app/models/nic/bmc.rb

```diff
@@ -12,15 +12,15 @@ def virtual
     end
     alias_method :virtual?, :virtual
 
-    attr_exportable :provider, :username, :password => ->(nic) { nic.decrypt_field(nic.password) }
+    attr_exportable :provider, :username, :password
```

Comment:
The code looks good, but I struggle to understand where the decrypt routine ended up now. The behaviour on this line was to immediately decrypt the field while in your patch `password` is an alias to `password_unredacted` under some conditions (or `nil` which is not relevant).

At which stage the password gets decrypted? 

Do we have a test coverage to assure the password is encrypted in the database and decrypted correctly when using this API?